src/trove_classifiers/__init__.py

```diff
@@ -768,4 +770,8 @@
 deprecated_classifiers = {
     "Natural Language :: Ukranian": ["Natural Language :: Ukrainian"],
     "Topic :: Communications :: Chat :: AOL Instant Messenger": [],
+    "License :: OSI Approved :: GNU Lesser General Public License v2 (LGPLv2)",
+    "License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)",
```

Comment:
This is a dict, these should be a mapping from a deprecated classifier name to a list of one or more classifiers that should be used in place of it.

Reply:
Okay, should I update. Anyways you told this is blocked.